tests/regression_tests/periodic/test.py

```diff
@@ -21,33 +21,46 @@ def _build_inputs(self):
         materials.export_to_xml()
 
         # Define geometry
-        x_min = openmc.XPlane(surface_id=1, x0=-5., boundary_type='periodic')
-        x_max = openmc.XPlane(surface_id=2, x0=5., boundary_type='periodic')
-        x_max.periodic_surface = x_min
+        x_min = openmc.XPlane(surface_id=1, x0=0., boundary_type='periodic')
```

Comment:
Since these are rotationally-periodic now, are we testing translational periodic still?

Reply:
There's a translational test along the z-axis